### PR TITLE
Prepare sample for inclusion in cloud docs

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
This will be used to update this Terraform example:

https://cloud.google.com/vpc/docs/using-vpc#create-custom-network

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
examples/simple_project/main.tf
--- old/examples/simple_project/main.tf
+++ new/examples/simple_project/main.tf
@@ -31,7 +31,7 @@
 # [START customvpc_create]
 module "test-vpc-module" {
   source       = "terraform-google-modules/cloud-router/google"
-  version = "~> 0.4"
+  version      = "~> 0.4"
   project_id   = var.project_id
   network_name = var.network_name
 
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

File: examples/simple_project/main.tf [hunk 1]
```diff
@@ -28,8 +28,10 @@ locals {
   subnet_03 = "${var.network_name}-subnet-03"
 }
 
+# [START customvpc_create]
```
Comment (morgante) [inline examples/simple_project/main.tf:31]: The region tag needs to use a valid product prefix: https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/code_snippets_101.md?cl=head

File: examples/simple_project/main.tf [hunk 2]
```diff
@@ -28,8 +28,10 @@ locals {
...
 module "test-vpc-module" {
-  source       = "../../"
+  source       = "terraform-google-modules/cloud-router/google"
```
Comment (morgante) [inline examples/simple_project/main.tf:33]: The source and version here should be based on this module, not Cloud Router.

**Review by morgante — CHANGES_REQUESTED** [review]

File: examples/simple_project/main.tf (hunk 2 above)
Comment (betsy-lichtenberg) [reply examples/simple_project/main.tf:33]: Got it

File: examples/simple_project/main.tf (hunk 1 above)
Comment (betsy-lichtenberg) [reply examples/simple_project/main.tf:31]: Got it.

File: examples/simple_project/main.tf [hunk 3]
```diff
@@ -28,10 +28,13 @@ locals {
...
 }
 
+# [START vpc_custom_create]
 module "test-vpc-module" {
-  source       = "../../"
+  source       = "terraform-google-modules/terraform-google-network/google"
+  version = "~> 3.2.0"
   project_id   = var.project_id
   network_name = var.network_name
+  mtu = var.mtu
```
Comment (morgante) [inline examples/simple_project/main.tf:37]: Variables need to be declared in variables.tf to be used. In this case, we should just hard code a value.

```suggestion
  mtu = 1460
```

File: examples/simple_project/main.tf [hunk 4]
```diff
@@ -28,10 +28,13 @@ locals {
   subnet_03 = "${var.network_name}-subnet-03"
 }
 
+# [START vpc_custom_create]
 module "test-vpc-module" {
-  source       = "../../"
+  source       = "terraform-google-modules/terraform-google-network/google"
+  version = "~> 3.2.0"
```
Comment (morgante) [inline examples/simple_project/main.tf:34]: Please run `terraform fmt`.

**Review by morgante — CHANGES_REQUESTED** [review]

File: examples/simple_project/main.tf (hunk 4 above)
Comment (betsy-lichtenberg) [reply examples/simple_project/main.tf]: Done. Thanks.

**Review by morgante — APPROVED** [review]

**Review by morgante — APPROVED** [review]

**Pull request merged**